nifi-nar-bundles/nifi-mqtt-bundle/nifi-mqtt-processors/src/main/java/org/apache/nifi/processors/mqtt/ConsumeMQTT.java

```diff
@@ -624,8 +619,8 @@ private void closeWriter(final RecordSetWriter writer) {
     }
 
     private String getTransitUri(String... appends) {
-        String broker = clientProperties.getBrokerUri().toString();
-        StringBuilder stringBuilder = new StringBuilder(broker.endsWith("/") ? broker : broker + "/");
+        final String broker = clientProperties.getRawBrokerUris();
```

Comment:
I don't think that the transitUri produced a valid URI in the original version, when more than one parameter were passed as an argument. It probably needs to be fixed, and we also need to think about how we want to represent a cluster URI here. Now its `ssl://localhost:1883,ssl://localhost:1884/` + `appends`, but I'm wondering if it would make sense to specify a cluster in the broker URI parameter as `ssl://[localhost:1883,localhost:1883]` since the scheme should be the same. This way, the same scheme validation could be spared.